content/PGP/Attestation.adoc

```diff
@@ -119,6 +119,14 @@ The pre-loaded attestation certificate is signed by a link:/PKI/yubico-ca-certs.
 NOTE: If you have a YubiKey Preview device, the attestation certificate will
 instead be signed by our link:/PKI/preview/yubico-preview-ca-certs.txt[Yubico OPGP Preview CA].
 
+=== Yubico Root CA Update
+
+Yubico is updating the root CA in 2025, and YubiKeys with firmware version 5.7.4 and later will be signed by our new root CA.  These certificates are linked to the link:/PKI/yubico-ca-certs.txt[Root Certificate Authority (Root CA)] through intermediate certificates, ensuring the integrity and trustworthiness of the YubiKey.
+
+* link:/PKI/yubico-ca-1.pem[Yubico Attestation Root 1]
+* link:/PKI/yubico-opgp-ca-1.pem[Yubico OpenPGP Attestation CA]
```

Comment:
We should swap the order of these in the description so it starts at Root, then does Intermediate, then the issuing CA. Sorry for not catching this in the Jira

Reply:
`yubico-ca-1.pem` and `yubico-opgp-ca-1.pem` both are root CAs (`opgp-ca-1` is the previous one from 2019, `ca-1` is the new root covering all features). Or did you mean something else?

Reply:
Nope you're correct. I just was reading too fast. Thanks. Ignore my original comment